stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -116,12 +114,23 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                 PaymentOptionResult.Failed(it)
             )
         }
+
+        // Enable animation for layout transitions
+        bottomSheet.layoutTransition.enableTransitionType(LayoutTransition.CHANGING)
+        fragmentContainerParent.layoutTransition.enableTransitionType(LayoutTransition.CHANGING)
+
         bottomSheetController.shouldFinish.observe(this) { shouldFinish ->
             if (shouldFinish) {
                 finish()
             }
         }
+
         bottomSheetController.setup()
+        // Fragment animates in immediately after activity is created, so we can already expand it
+        bottomSheetController.expand()
+        // isFitToContents causes conflicts when calculating the sheet position upon resize.
+        // CoordinatorLayout will already position the sheet correctly with gravity = bottom.
+        bottomSheetBehavior.isFitToContents = false
```

Comment:
Can we move any of this logic into the base activity?

Reply:
Yes, some of it. Done.

Reply:
Yes, some of it. Done.